### PR TITLE
[ADD] website_sale_search_no_kept: New module to delete product search criteria when the user clicks to other e-commerce category

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa

### CODE REVIEW

File: website_sale_search_no_kept/README.rst [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/e-commerce-11-0/e-commerce-11-0-website_sale_search_no_kept
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/113/11.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module extends the functionality of website sale module to allow to no
```
Comment (pedrobaeza) [inline website_sale_search_no_kept/README.rst]: s/no/not

File: website_sale_search_no_kept/README.rst [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+preserve search content when user navigate to other e-commerce category.
```
Comment (pedrobaeza) [inline website_sale_search_no_kept/README.rst]: s/navigate/navigates

File: website_sale_search_no_kept/README.rst [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+When you search products by keyword the search criterion is maintained, so when
```
Comment (pedrobaeza) [inline website_sale_search_no_kept/README.rst]: comma after `keyword`

s/criteria/criterion

File: website_sale_search_no_kept/README.rst [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+you click to other category only get the products that match the search.
```
Comment (pedrobaeza) [inline website_sale_search_no_kept/README.rst]: s/only/, you only

File: website_sale_search_no_kept/README.rst [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+.. contents::
+   :local:
+
+Usage
+=====
+
+#. Go to Website Shop.
+#. Input a text to search products.
+#. Click to other category.
+#. You will see all category products.
```
Comment (pedrobaeza) [inline website_sale_search_no_kept/README.rst]: add `without filtering by the input text`

**Review by pedrobaeza — APPROVED** [review]

**Comment by sergio-teruel** [comment]
@pedrobaeza changes done!!

**Review by martintorresvm — APPROVED** [review]
LGTM and tested on Runbot

**Pull request merged**